podpac/datalib/soilgrids.py

```diff
@@ -0,0 +1,119 @@
+"""
+SoilGrids
+
+See: https://maps.isric.org/
+"""
+
+
+from podpac.data import WCS
+
+
+class SoilGridsBase(WCS):
+    """ Base SoilGrids WCS datasource. """
+
+    format = "geotiff_byte"
+    max_size = 16384
+    _repr_keys = ["layer"]
+
+
+class SoilGridsWRB(SoilGridsBase):
+    """ SoilGrids: WRB classes and probabilities (WCS) """
+
+    source = "https://maps.isric.org/mapserv?map=/map/wrb.map"
```

Comment:
In terms of an implementation for a datasource, that is gloriously simple. Love it.